Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -260,8 +289,11 @@ public void CLIDynamicJSON_ExistingCustomized_IDsAndNames_Test()
 
             // Act            
             CLIProcessor CLI = new CLIProcessor();
-            CLI.ExecuteArgs(new string[] { "dynamic", "-f", jsonConfigFilePath });      
-
+            
+            Task.Run(async () =>
+            {
+                await CLI.ExecuteArgs(new string[] { "dynamic", "-f", jsonConfigFilePath });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)